docs/csharp/whats-new/csharp-7-3.md

```diff
@@ -45,68 +45,90 @@ unsafe struct S
-var arr2 = new int[] { 1, 2, 3};
+var arr2 = new int[] {1, 2, 3};
 ```
 
 Now, that same syntax can be applied to arrays that are declared with `stackalloc`:
 
 ```csharp
 int* pArr = stackalloc int[3] {1, 2, 3};
-int* pArr2 = stackalloc int[] { 1, 2, 3};
+int* pArr2 = stackalloc int[] {1, 2, 3};
 Span<int> arr = stackalloc [] {1, 2, 3};
 ```
 
-You can learn more in the article on the [`stackalloc` statement](../language-reference/keywords/stackalloc.md).
+For more information, see the [`stackalloc` statement](../language-reference/keywords/stackalloc.md) article in the language reference.
 
 ### More types support the `fixed` statement
 
 The `fixed` statement supported a limited set of types. Starting with C# 7.3, any type that contains a `DangerousGetPinnableReference()` method that returns a `ref T` or `ref readonly T` may be `fixed`. Adding this feature means that `fixed` can be used with <xref:System.Span%601?displayProperty=nameWithType> and related types.
 
-You can learn more in the article on the [`fixed` statement](../language-reference/keywords/fixed-statement.md).
+For more information, see the [`fixed` statement](../language-reference/keywords/fixed-statement.md) article in the language reference.
 
 ### Enhanced generic constraints
 
 You can now specify the type <xref:System.Enum?displayProperty=nameWithType> or <xref:System.Delegate?displayProperty=nameWithType> as base class constraints for a type parameter.
 
-You can also use the new `unmanaged` constraint, to specify that a type parameter must be an **unmanaged type**. An **unmanaged type** is a type that isn't a reference type, and doesn't contain any reference type at any level of nesting.
+You can also use the new `unmanaged` constraint, to specify that a type parameter must be an **unmanaged type**. An **unmanaged type** is a type that isn't a reference type and doesn't contain any reference type at any level of nesting.
 
-You can learn more in the articles on [`where` generic constraints](../language-reference/keywords/where-generic-type-constraints.md) and the article covering [constraints on type parameters](../programming-guide/generics/constraints-on-type-parameters.md).
+For more information, see the articles on [`where` generic constraints](../language-reference/keywords/where-generic-type-constraint.md) and [constraints on type parameters](../programming-guide/generics/constraints-on-type-parameters.md).
 
 ## Make existing features better
 
 The second theme provides improvements to features in the language. These features improve productivity when writing C#.
 
 ### Tuples support `==` and `!=`
 
-The C# tuple types now support `==` and `!=`. You can learn more about the rules in the article on [tuples](../tuples.md#equality-and-tuples).
+The C# tuple types now support `==` and `!=`. Fore more information, see the section covering [equality](../tuples.md#equality-and-tuples) in the article on [tuples](../tuples.md#equality-and-tuples).
```

Comment:
nit: the sentence should end with the link to the article itself, not the section:
`...in the article on [tuples](../tuples.md)`